client/my-sites/backup/backup-contents-page/file-browser/file-browser-node.tsx

```diff
@@ -83,7 +83,7 @@ const FileBrowserNode: FunctionComponent< FileBrowserNodeProps > = ( {
 	};
 
 	const nodeClassName = classNames( 'file-browser-node', { 'is-alternate': isAlternate } );
-	const [ label, isLabelTruncated ] = useTruncatedFileName( item.name, 30 );
+	const [ label, isLabelTruncated ] = useTruncatedFileName( item.name, 30, item.type );
```

Comment:
Maybe we should consider to expand the chars limit to 45 instead of 30. This way a string like `original version 15.10.10 from WordPress.org` will not be truncated and we don't need to add some hack for that.